torch/csrc/api/include/torch/nn/functional.h

```diff
@@ -9,3 +9,4 @@
 #include <torch/nn/functional/pixelshuffle.h>
 #include <torch/nn/functional/pooling.h>
 #include <torch/nn/functional/vision.h>
+#include <torch/nn/functional/dropout.h>
```

Comment:
nit: would be awesome to put this line above `#include <torch/nn/functional/embedding.h>`, following alphabetical order